especifico/json_schema.py

```diff
@@ -93,13 +93,22 @@ def _do_resolve(node):
                     return resolved
         elif isinstance(node, Mapping):
```

Comment:
since we are already changing this code heavily, can you please optimize it a little bit
```
if isinstance(node, Mapping):
    if "$ref" in node:
        ...
    else:
        # do not resolve refs in jsonschema nested in the spec
        ...
else:
    ...
```